haiku-data/yab_documentation/yab_documentation-1.7.8.recipe

```diff
@@ -0,0 +1,30 @@
+SUMMARY="An integrated development environment for yab"
+DESCRIPTION="Documentaion for yab"
+HOMEPAGE="http://software.besly.de/"
+COPYRIGHT="2006-2015 Jan Bungeroth
+	2015-2017 Jim Saxton
+	2020 Lorenz Glaser (lorglas) BeSly Software Solutions"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://github.com/lorglas/yab_documentation/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="b6dbd04d4eadaba199c80d71f803b682727fb35ed4908bd17c7267677d9e5549"
+#SOURCE_FILENAME="yab_documentation-$portVersion.tar.gz"
```

Comment:
You can enable this, looks better to have something meaningful instead of a $protVersion.tar.gz
``diff
- #SOURCE_FILENAME="yab_documentation-$portVersion.tar.gz"
+ SOURCE_FILENAME="yab_documentation-$portVersion.tar.gz"
```